src/ucp/core/ucp_mm.c

```diff
@@ -718,16 +719,19 @@ ucp_memh_init_from_parent(ucp_mem_h memh, ucp_md_map_t parent_md_map)
 }
 
 static ucs_status_t ucp_memh_init_uct_reg(ucp_context_h context, ucp_mem_h memh,
-                                          unsigned uct_flags,
+                                          unsigned uct_flags, ucp_md_map_t md_map,
```

Comment:
maybe we could use UCT_MD_MEM_FLAG_LOCK in uct_flags for rndv_frag memory pools, and this would also allow using MDs that support only blocking registration?

Reply:
fixed in place as the patch is small